lib/src/controls/flyouts/menu.dart

```diff
@@ -0,0 +1,532 @@
+import 'dart:async';
+
+import 'package:fluent_ui/fluent_ui.dart';
+import 'package:flutter/foundation.dart';
+
+/// Menu flyouts are used in menu and context menu scenarios to display a list
+/// of commands or options when requested by the user. A menu flyout shows a
+/// single, inline, top-level menu that can have menu items and sub-menus.
+///
+/// ![MenuFlyout](https://docs.microsoft.com/en-us/windows/apps/design/controls/images/contextmenu_rs2_icons.png)
+///
+/// See also:
+///
+///  * [FlyoutController], which displays a flyout to the given target
+///  * <https://learn.microsoft.com/en-us/windows/apps/design/controls/menus>
+class MenuFlyout extends StatefulWidget {
+  /// Creates a menu flyout.
+  const MenuFlyout({
+    Key? key,
+    this.items = const [],
+    this.color,
+    this.shape,
+    this.shadowColor = Colors.black,
+    this.elevation = 8.0,
+    this.constraints,
+    this.padding = const EdgeInsets.only(top: 8.0),
+  }) : super(key: key);
+
+  /// {@template fluent_ui.flyouts.menu.items}
+  /// The colletion used to generate the content of the menu.
```

Comment:
Maybe be more esplicative on how this is used? For example could specify this is a list and also that items are arranged in a vertical list